tests/functional/collection/test_collection_delete.py

```diff
@@ -1,42 +1,52 @@
+import pytest
 from globus_sdk._testing import load_response_set
 
 
-def test_guest_collection_delete(run_line, add_gcs_login):
+# toggle between the (newer) 'gcs' variant and the 'bare' variant
+@pytest.fixture(params=(True, False), ids=("gcs-collection", "collection"))
+def base_command(request):
+    if request.param:
+        return ["globus", "gcs", "collection", "delete"]
+    else:
+        return ["globus", "collection", "delete"]
```

Comment:
That structure for the params is better, but I don't agree about moving off of lists for command passing to `run_line` as a rule. I think we should continue to support both modes of use in the testsuite, and use whichever one is more appropriate to the circumstance.

Dynamically building arguments is more easily defined as list manipulation, and I think it's important that we be able to directly pass lists so that we can precisely control the strings passed for parameters without having to reason about `shlex` behaviors.
I think the `shlex` usage is an unnecessary potential source of error when we are testing behaviors regarding quotes embedded in strings, literal JSON data on the command line, etc.

So I'll implement something like the change suggested -- I agree it's a good step not to pass a bool if possible! -- but I won't be turning things into f-strings.